docs/docs/api.rst

```diff
@@ -47,7 +47,7 @@ Hilbert
    netket.hilbert.Spin
    netket.hilbert.Qubit
    netket.hilbert.Fock
-   netket.hilbert.ContinuousBoson
+   netket.hilbert.Particle
 
```

Comment:
you probably should add `ContinuousOperator`, `Kinetic` and `Potential` to the operator section below. Maybe create a new subsection called `Continuous space operators`